I0328 22:41:08.686053       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5308", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Role/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 22:41:08.689745       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5308", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ServiceAccount/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 22:41:08.701026       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5308", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create RoleBinding/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 22:41:08.705253       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5308", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Deployment/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 22:41:08.710665       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5308", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Service/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 22:41:08.717590       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5308", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd in  test-cluster successful
I0328 22:41:08.721581       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5308", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd-peer in  test-cluster successful
I0328 22:41:08.728164       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5308", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-pd-3731616 for controller TidbCluster/test-cluster successfully
I0328 22:41:08.735798       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5308", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-pd in  test-cluster successful
I0328 22:41:08.747304       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:41:08.747338       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 22:41:08.804885       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.132.224:2379: connect: network is unreachable, service acto-namespace/test-cluster-pd has no endpoints
I0328 22:41:08.846476       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 22:41:08.846517       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 22:41:08.910798       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.207.192:2379: connect: network is unreachable, service acto-namespace/test-cluster-pd has no endpoints
E0328 22:41:08.919631       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 22:41:09.812523       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.207.192:2379: connect: network is unreachable, service acto-namespace/test-cluster-pd has no endpoints
E0328 22:41:09.821479       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 22:41:13.603400       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.207.192:2379: connect: network is unreachable, service acto-namespace/test-cluster-pd has no endpoints
E0328 22:41:13.614878       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 22:41:18.896855       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.132.224:2379: connect: connection refused, service acto-namespace/test-cluster-pd has no endpoints
E0328 22:41:18.908338       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 22:41:30.477046       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.132.224:2379: connect: connection refused
I0328 22:41:30.504976       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 22:41:30.505029       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0328 22:41:30.567698       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.132.224:2379: connect: connection refused
I0328 22:41:30.609513       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 22:41:30.609571       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0328 22:41:30.668511       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.132.224:2379: connect: connection refused
I0328 22:41:30.715677       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 22:41:30.715726       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0328 22:41:30.780030       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.132.224:2379: connect: connection refused
E0328 22:41:30.788025       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0328 22:41:30.959968       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.132.224:2379: connect: connection refused
I0328 22:41:30.983398       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 22:41:30.983448       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0328 22:41:31.560113       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.132.224:2379: connect: connection refused
E0328 22:41:31.570523       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0328 22:41:43.602837       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
E0328 22:41:43.611591       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
I0328 22:41:51.011014       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5525", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tikv-peer in  test-cluster successful
I0328 22:41:51.014489       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5525", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tikv-3831336 for controller TidbCluster/test-cluster successfully
I0328 22:41:51.020901       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5525", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tikv in  test-cluster successful
I0328 22:41:51.031625       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:41:51.031662       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0328 22:41:51.161212       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:41:51.161263       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0328 22:41:51.306727       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:41:51.306772       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0328 22:41:52.006827       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0328 22:41:52.807933       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0328 22:41:53.628915       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:41:53.628956       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0328 22:41:54.459793       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:41:54.459845       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0328 22:41:55.205772       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0328 22:42:13.649393       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5661", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb-peer in  test-cluster successful
I0328 22:42:13.660503       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5661", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb in  test-cluster successful
I0328 22:42:13.664931       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5661", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tidb-6662316 for controller TidbCluster/test-cluster successfully
I0328 22:42:13.672094       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"2c2cb5b6-4552-409f-9a69-7f06609cde55", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"5661", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tidb in  test-cluster successful
I0328 22:42:14.679362       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:42:14.781795       1 equality.go:87] Service spec diff for acto-namespace/test-cluster-tidb:   v1.ServiceSpec{
  	Ports: []v1.ServicePort{
  		{
  			... // 3 identical fields
  			Port:       4000,
  			TargetPort: {IntVal: 4000},
- 			NodePort:   0,
+ 			NodePort:   32729,
  		},
  		{
  			... // 3 identical fields
  			Port:       10080,
  			TargetPort: {IntVal: 10080},
- 			NodePort:   0,
+ 			NodePort:   32397,
  		},
  	},
  	Selector:  {"app.kubernetes.io/component": "tidb", "app.kubernetes.io/instance": "test-cluster", "app.kubernetes.io/managed-by": "tidb-operator", "app.kubernetes.io/name": "tidb-cluster"},
  	ClusterIP: "",
  	... // 12 identical fields
  }
I0328 22:42:14.781850       1 tidb_member_manager.go:442] Sync TiDB service acto-namespace/test-cluster-tidb, spec equal: false, annotations equal: true, label equal: true
I0328 22:42:14.852257       1 service_control.go:88] update Service: [acto-namespace/test-cluster-tidb] successfully, kind: , name: test-cluster
I0328 22:42:14.875835       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
E0328 22:42:15.255914       1 pod_control.go:185] failed to update pod acto-namespace/test-cluster-tidb-1 with cluster labels map[app.kubernetes.io/component:tidb app.kubernetes.io/instance:test-cluster app.kubernetes.io/managed-by:tidb-operator app.kubernetes.io/name:tidb-cluster controller-revision-hash:test-cluster-tidb-59fc8d598d statefulset.kubernetes.io/pod-name:test-cluster-tidb-1 tidb.pingcap.com/cluster-id:7215726631408793568], TidbCluster: test-cluster, err: Operation cannot be fulfilled on pods "test-cluster-tidb-1": the object has been modified; please apply your changes to the latest version and try again
E0328 22:42:15.655758       1 pod_control.go:185] failed to update pod acto-namespace/test-cluster-tidb-2 with cluster labels map[app.kubernetes.io/component:tidb app.kubernetes.io/instance:test-cluster app.kubernetes.io/managed-by:tidb-operator app.kubernetes.io/name:tidb-cluster controller-revision-hash:test-cluster-tidb-59fc8d598d statefulset.kubernetes.io/pod-name:test-cluster-tidb-2 tidb.pingcap.com/cluster-id:7215726631408793568], TidbCluster: test-cluster, err: Operation cannot be fulfilled on pods "test-cluster-tidb-2": the object has been modified; please apply your changes to the latest version and try again
I0328 22:42:15.876336       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:42:15.998349       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 22:42:33.854518       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 22:42:33.878423       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:42:34.211902       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 22:42:34.262172       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 22:42:35.411755       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 22:42:43.658554       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped

